### DISCUSSION TITLE
Is it possible not to log / count toward stats a domain, or filter it when querying?

### DISCUSSION THREAD

**Discussion by KaoDome** [AdguardTeam/AdGuardHome#6117] [Q&A]
Hi, I was wondering it any of that was possible, I was thinking of opening making it a feature request in the issues section, but I wanted to know if it was already possible. I tried looking through the opened issues and discussions here, but it is crazy, I don't really know how you guys prioritize or know if something was already proposed 😅

So, basically I have a couple of domains that have several orders of magnitude more requests than the rest and are expected to show up in logs or queries (e.g., www.google.com), I would like not to include those queries in logs directly, or perhaps be able to filter them in the statistics UI so they aren't counted and eliminate them when querying the log.

Is it possible right now? Has it been proposed before?

**Comment by ainar-g** [comment]
Do you mean the *Ignored domains* sections on the *Settings → General settings* page?
